runelite-client/src/main/java/net/runelite/client/plugins/dpscounter/DpsCounterPlugin.java

```diff
@@ -181,14 +220,21 @@ public void onHitsplatApplied(HitsplatApplied hitsplatApplied)
 					specialCounterUpdate.setMemberId(localMember.getMemberId());
 					wsClient.send(specialCounterUpdate);
 				}
+				// damage to specific targets
+				if (dpsConfig.targetDamage()) {
+					if (!targets.containsKey(npcName)) {
+						targets.put(npcName, hitsplat.getAmount());
+					} else {
```

Comment:
Need to change these to meet the style guide.

```suggestion
				if (dpsConfig.targetDamage())
				{
					if (!targets.containsKey(npcName))
					{
						targets.put(npcName, hitsplat.getAmount());
					}
					else
					{
```